app/controllers/catalog_controller.rb

```diff
@@ -222,7 +222,13 @@ def show
                                                     versions: object_client.version.inventory)
 
     @datastreams = object_client.metadata.datastreams
-    @techmd = TechmdService.techmd_for(druid: params[:id])
+
+    # If you have this token, it indicates you have view_metadata access to the object
+    @verified_token_with_expiration = Argo.verifier.generate(
+      { key: params[:id] },
+      expires_in: 5.minutes,
```

Comment:
I'd make this longer, like 30/60 minutes?

Reply:
Good idea. I made it an hour.